tests/config.tf

```diff
@@ -0,0 +1,43 @@
+locals {
```

Comment:
have the test terraform setups in their separate folder under "./tests" directory, set name of the test like, "simple-setup" , "full-setup" or "my-special-case-with-custom-option-b"